x-pack/plugins/xpack_main/server/lib/telemetry/monitoring/get_high_level_stats.js

```diff
@@ -187,8 +188,75 @@ export function getHighLevelStats(server, callCluster, clusterUuids, start, end,
  * @param {String} product The product to limit too ('kibana', 'logstash', 'beats')
  * @return {Promise} Response for the instances to fetch detailed for the product.
  */
-export function fetchHighLevelStats(server, callCluster, clusterUuids, start, end, product) {
+
+
+export function getLegacyParams(server, clusterUuids, start, end, product) {
   const config = server.config();
+
+  return {
+    index: getIndexPatternForStackProduct(product),
+    size: config.get('xpack.monitoring.max_bucket_size'),
+    ignoreUnavailable: true,
+    filterPath: [
+      'hits.hits._source.cluster_uuid',
+      `hits.hits._source.${product}_stats.${product}.version`,
+      `hits.hits._source.${product}_stats.os`,
+      `hits.hits._source.${product}_stats.usage`,
+      // we don't want metadata
+      `hits.hits._source.${product}_stats.cloud.name`,
+      `hits.hits._source.${product}_stats.cloud.id`,
+      `hits.hits._source.${product}_stats.cloud.vm_type`,
+      `hits.hits._source.${product}_stats.cloud.region`,
+      `hits.hits._source.${product}_stats.cloud.zone`
+    ],
+    body: {
+      query: createQuery({
+        start,
+        end,
+        type: `${product}_stats`,
+        filters: [
+          { terms: { cluster_uuid: clusterUuids } },
+          {
+            'bool': {
+              'must': [
+                { 'match': { 'kibana_stats.kibana.statuses.name': 'includes-usage-data' } },
```

Comment:
Rather than a separate query,  I think this just be part of the filter when we call `fetchHigherLevelStats`?  If the filter is OR clause it can match "new" or "legacy".  And then as a fallback if it matches none we can query with no filters at all, and assume it is a cluster that is still being updated with usage every 10s.